devicetypes/smartthings/inovelli-dimmer.src/inovelli-dimmer.groovy

```diff
@@ -0,0 +1,515 @@
+/*	Copyright 2020 SmartThings
+*
+*	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+*	in compliance with the License. You may obtain a copy of the License at:
+*
+*		http://www.apache.org/licenses/LICENSE-2.0
+*
+*	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+*	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+*	for the specific language governing permissions and limitations under the License.
+*
+*	Inovelli Dimmer
+*
+*	Copyright 2020 SmartThings
+*
+*/
+metadata {
+	definition(name: "Inovelli Dimmer", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch", minHubCoreVersion: '000.017.0012', mcdSync: true) {
+		capability "Actuator"
+		capability "Configuration"
+		capability "Energy Meter"
+		capability "Health Check"
+		capability "Refresh"
+		capability "Sensor"
+		capability "Switch"
+		capability "Switch Level"
+		capability "Power Meter"
+
+		fingerprint mfr: "031E", prod: "0001", model: "0001", deviceJoinName: "Inovelli Dimmer Switch" , vid: "SmartThings-smartthings-Inovelli_Dimmer"//Inovelli Dimmer LZW31-SN
```

Comment:
shouldn't this metadata be picked up automatically? And I think you'd need to specify an `mnmn` anyway if it weren't.

Reply:
Done